### DISCUSSION TITLE
Add a question to photograph a document or document to the form

### DISCUSSION THREAD

**Discussion by zakoornew** [community/community#61550] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a suggestion for you regarding the photo question
Please add a document or paper image question to the form
So that the application automatically selects the paper or document
Automatically defines the edges of the paper or document
We are very suffering in this regard
Thank you for this wonderful application and we wish you continued development and progress
All thanks and appreciation to you

**Comment by queenofcorgis** [comment]
Hi there!  So we can properly categorize your feedback, which form is this regarding?
